nix/web-security-tracker.nix

```
@@ -105,6 +106,12 @@ in
       type = types.attrsOf types.path;
       default = { };
     };
+    cve.startDate = mkOption {
+      type = types.nullOr types.str;
+      default = null;
+      defaultText = "the application default: last current year";
```

```suggestion
      defaultText = "the application default: January 1st of the prior year";
```

Regarding "why", maybe the option's `description` the right place to put it because ideally this is the only place where anyone would ever have to manually interact with it.